integration-tests/src/test/kotlin/org/stellar/anchor/platform/TestAccounts.kt

```diff
@@ -1,10 +1,6 @@
 package org.stellar.anchor.platform
 
-var CLIENT_WALLET_SECRET = "SCQ2YRUGKBO4Q6RWSXV3SYVGHJB525XVKUP2HDGVW3LV42UBLGTZ24XX"
-var CLIENT_WALLET_ACCOUNT = "GCHU3RZAECOKGM2YAJLQIIYB2ZPLMFTTGN5D3XZNX4RDOEERVLXO7HU4"
+var CLIENT_WALLET_SECRET = "SBHTWEF5U7FK53FLGDMBQYGXRUJ24VBM3M6VDXCHRIGCRG3Z64PH45LW"
+var CLIENT_WALLET_ACCOUNT = "GDJLBYYKMCXNVVNABOE66NYXQGIA5AC5D223Z2KF6ZEYK4UBCA7FKLTG"
 var CLIENT_WALLET_EXTRA_SIGNER_1_SECRET = "SDUDRKCL5AX7RDZ7S6JAPBNKLV6LRZXJSHN5OJDP32TIJB42ODPQODHY"
-var CLIENT_WALLET_EXTRA_SIGNER_1_ACCOUNT =
-  "GC6X2ANA2OS3O2ESHUV6X44NH6J46EP2EO2JB7563Y7DYOIXFKHMHJ5O"
```

Comment:
Are these not used?

Reply:
No they are not used. They are signer accounts used for testing. Not for transactions.